Kitodo-DataEditor/src/test/resources/kitodo_config.properties

```diff
@@ -13,3 +13,4 @@ MetsEditorDefaultPagination=uncounted
 directory.modules=../Kitodo/modules/
 xsltFilenameMetadataTransformation=MetsModsGoobi_to_MetsKitodo.xsl
 directory.metadata=../../../../src/test/resources/metadata/
+script_createDirMeta=src/test/resources/scripts/script_createDirMeta.sh
```

Comment:
I think it's only needed in Kitodo-FileManagement where you also added it, istn't it?

Reply:
I thought the same thing at first, but as you can see from [this Travis run](https://travis-ci.org/github/kitodo/kitodo-production/builds/716495360) (that was right before I added it here), several tests require it.